rezerwacje-front/src/views/Users/components/UsersForm/UsersForm.js

```diff
@@ -0,0 +1,276 @@
+import React from 'react';
+import {
+  Card,
+  CardHeader,
+  CardContent,
+  CardActions,
+  Divider,
+  Grid,
+  Button,
+  TextField,
+  Typography,
+  Radio, 
+  RadioGroup,
+  FormControlLabel
+} from '@material-ui/core';
+import useForm from "./useForm"
+import validateUserForm from "./validateUserForm"
+import './Users.scss'
+
+const UsersForm = () => {
+
+   const initState = {
+    firstName: '',
+    lastName: '',
+    email: '',
+    phone: '',
+    login: '',
+    password: '',
+    confirmPassword: '',
+    role: ''
+  };
+
+
+  const submit = () => {
+
+      const {firstName, lastName, email, phone, login, password, role} = values
+      console.log(JSON.stringify({firstName: firstName, lastName: lastName, emailAddress: email, phoneNumber: phone, 
+        login: login,  password: password, userType: role}))
+      
+        fetch("/api/system-user", {
+          method: "POST", 
+          headers: { 'Content-Type': 'application/json' },
+          body: JSON.stringify({firstName: firstName, lastName: lastName, emailAddress: email, phoneNumber: phone, 
+      login: login,  password: password, userType: role}),
+        }).then(
```

Comment:
Messy indentation.  
BTW Visual Studio Code does wonders to code formatting on Ctrl+S

Reply:
It does if you have "format on save" on. Otherwise, the shortcut is Ctrl+Shift+I